models/remoteworkitem_test.go

```diff
@@ -0,0 +1,92 @@
+// +build unit
+
+package models
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"net/http"
+	"testing"
+)
+
+func TestMap(t *testing.T) {
+
+	// This is the Type to be converted to
+	workItemType := wellKnown["1"] // Will remove this and add an actual Type.
+
+	remoteWorkItem := RemoteWorkItem{
+		Fields: map[string]interface{}{
+			"id":          "3",
+			"description": "I am a remote issue's description in Github",
+			"title":       "I am the title of the issue",
```

Comment:
Should hide one of the elements under a parent to verify that you can reach down into the format. e.g. 

```
{
 "id": "3",
 "meta": {
   "descrpition": "a",
   "title": "b"
 }
}
```